packages/estatico-webpack/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@unic/estatico-webpack",
-  "version": "0.0.20",
+  "version": "1.0.0",
```

i'd argue it makes more sense having this added under a patch/bugfix release, as it doesn't break or add anything in the api

```suggestion
  "version": "0.0.21",
```